.buildkite/pipelines/pull_request/base.yml

```diff
@@ -69,6 +69,19 @@ steps:
         - exit_status: '-1'
           limit: 3
 
+  - command: SERVERLESS_ENVIRONMENT=observability.examples .buildkite/scripts/steps/functional/serverless_ftr.sh
```

Comment:
Do we want these to run on merge too?  If so there's another pipeline setup at `.buildkite/pipelines/serverless.yml`